Channels/Integration/01e_Magento2.md

```diff
@@ -0,0 +1,167 @@
+# Manage the Magento 2 connection
+
+To establish a connection to a Magento 2 shop, there are several particularities to consider. Therefore, the configuration of the Magento 2 connection is described in detail below.
```

Comment:
Warum haben wir hier nicht das Erstellen der Verbindung beschrieben? Brauche nochmal ne Erinnerung...

Reply:
URL, Username und Password nötig. Nicht weiter erklärt in Wissenstransfer.